Remake by rick 2

0x07CB opened the pull request:
From branch: remake-by-rick-1
with:

  - [ + ] Update the install script to install dependencies
  - [ + ] change colors of the banner
  - [ + ] add base64 encoding choice before build the command
  - [ + ] test if iptables is installed and try to open port
  - [ + ] close the program if test _specified_port_inbound_firewall_localhost have an error


the remake number 2 include:
- [ + ] Fernet encryption mode with key generation into an file to /tmp/
- [ + ] Message on terminal with the path of the fernet key (random name for this file)
- [ + ] Compression with gzip , bzip2 , lzma 
- [ + ] Automatic enabling of base64 if encryption or compression enabled
- [ + ] Fernet option code use smyley symbols 🔑  and 💣 into terminal

I feel upgrade more this tools can be very fun ! 👍🏻 You have made an good code and I hope you can appreciate my modifications.

TTYL,
Rick Sanchez

0x07CB committed:
update the install, add an script to install dependancies

0x07CB committed:
change colors of the banner

0x07CB committed:
add base64 encoding choice of the command

0x07CB committed:
add base64 encoding choice of the command

0x07CB committed:
finish test_specified_port_inbound_firewall_localhost

0x07CB committed:
add more control for the firewall port opening

0x07CB committed:
add choice to auto manage iptables or not

0x07CB committed:
add fernet encryption option and gzip, bzip and lzma compression options

0x07CB committed:
add .vscode

0x07CB committed:
remove unused libs imported

0x07CB committed:
fixing command decoding at the end(I have forgot) if is bytes to be copied into clipboard and good

0x07CB committed:
update the install.sh

t0thkr1s commented:
Hi!

Thanks for creating a pull a request! I need some time to review your code and test it. I really appreciate the work you have done, keep it up!

0x07CB commented:
Cool so, I continue to up changes.
😉🚀

0x07CB commented:
> Hi!
> 
> Thanks for creating a pull a request! I need some time to review your code and test it. I really appreciate the work you have done, keep it up!

I have made recently an password dictionary attack for protected ZIP files, @t0thkr1s  just look: [zipBruter](https://github.com/0x07CB/zip-bruter/blob/main/README.md)
It's very light, small code.
and...
It's write in python3. ;)
So, What do you think about this code ?

TTYL,
_Rick Sanchez_

t0thkr1s committed:
delete settings.json

I'd rather keep the repository clean and simple. Dev environment related files aren't needed for others.

t0thkr1s commented:
I have managed to test your code and it looks good for me. I'll definitely make some very small changes (not related to the logic) just to make tool's output consistent. I already deleted the VSCode settings.json file, I believe that's not useful for users and they don't need to download it every time they need the tool. I'll try to keep the code and functionality simple that's the intention behind this tool. Anyway, keep up the good work! I'll merge your changes as fast as I possibly can! Cheers 🎉
